plugin.json

```diff
@@ -2,10 +2,34 @@
     "name": "PostHog Heartbeat Plugin",
     "url": "https://github.com/PostHog/posthog-heartbeat-plugin",
     "description": "Creates one event per minute to validate ingestion pipeline",
-    "main": "index.js",
     "config": [
         {
-            "markdown": "This plugin triggers an event called heartbeat every minute"
+            "markdown": "Events are emitted once per minute."
+        },
+        {
+            "key": "events",
+            "name": "Events to be emitted",
+            "type": "choice",
+            "hint": "The basic `heartbeat` event emitted by this plugin goes straight into the event ingestion queue. The optional `heartbeat_api` event goes via the event capture endpoint, which helps monitor the API too.",
+            "choices": [
+                "heartbeat",
+                "heartbeat + heartbeat_api"
+            ],
+            "default": "heartbeat",
+            "required": true
+        },
```

Comment:
Lets make this two separate boolean inputs that you can enable independently or both, making it a more user friendly experience :)

Reply:
discussed it's not an option to use checkboxes atm, though it would be nice if we extended the types